packaging/chocolatey/crc/update.ps1

```diff
@@ -0,0 +1,36 @@
+import-module au
+
+$releases = 'https://github.com/crc-org/crc/releases/tag/'
+
+function global:au_SearchReplace {
+   @{
+        ".\crc.nuspec" = @{
+            "\<version\>.+" = "<version>$($Latest.Version)</version>"
+            "\<releaseNotes\>.+" = "<releaseNotes>$($Latest.ReleaseNotes)</releaseNotes>"
+        }
+    }
+}
+
+function global:au_GetLatest {
+    $crcVersion = (Invoke-WebRequest https://api.github.com/repos/crc-org/crc/tags | Select-Object -ExpandProperty Content | ConvertFrom-Json)[0].name
+    @{
+        Version = $crcVersion.Remove(0,1)
+        ReleaseNotes = $releases + $crcVersion
+    }
+}
+
+function global:au_BeforeUpdate {
+    # download admin helper release
+    $admin_helper_version = (Invoke-WebRequest https://api.github.com/repos/crc-org/admin-helper/tags | Select-Object -ExpandProperty Content | ConvertFrom-Json)[0].name
+    $Latest.URL64 = 'https://github.com/crc-org/admin-helper/releases/download/' + $admin_helper_version + '/crc-admin-helper-windows.exe'

```

Comment:
only issue is it downloads the `tray` zip along with the `admin-helper` binary, currently in the makefile, downloaded tray zip is removed after running `crc-embedder` and before running `choco pack` to avoid including it in the finale choco pkg. 

created https://github.com/crc-org/crc/pull/3475 to be able to select only the admin-helper to download

(we could also point crc-embedder to download to a temporary directory and we only copy the admin-helper binary to the tools/ directory)